### ISSUE TITLE
Problems displaying the screen when connecting directly ("--connect ID" parameter).

### ISSUE THREAD

**Issue by Jemmota** [rustdesk/rustdesk#5791]
### Bug Description

Note the strange behavior of RustDesk when we directly connect using the "--connect ID" parameter. Two problems occur in the most recent version (1.2.3 Nightly): 1) The display of white screens at different points on the screen before displaying the final connection screen (black screen); 2) The displacement of the window positioning with each new call/display of the connection window. In version 1.2.2 (stable) only the second problem occurs.

### How to Reproduce

Create a RustDesk.exe link and in the link properties add the "--connect ID" parameter (for example "c:\Rustdesk.exe --connect 1234567890"

### Expected Behavior

It will display a blank screen twice. The first in the top left corner; the second in the middle of the screen (causing an unpleasant blinking effect), and only after that it displays (on the third display) the default connection screen in the correct position in which the window was saved during the last access. However, after successive calls the connection screen is moved a few pixels to the right and down.

### Operating system(s) on local side and remote side

Windows 11

### RustDesk Version(s) on local side and remote side

1.2.2 (Stable) and 1.2.3 (Nightly) 

### Screenshots

https://github.com/rustdesk/rustdesk/assets/129885668/5e8411b1-baf1-4d7b-bb94-ebf2ddcefd38

**Comment by rustdesk** [comment]
@fufesou

**Comment by fufesou** [comment]
Thanks for your feedback.

>  blank screen
I can reproduce. I'll try to fix it.

> screen is moved a few pixels to the right and down
I cannot reproduce.

For the second issue, the position may be moved
https://github.com/rustdesk/rustdesk/blob/master/flutter/lib/common.dart#L1674

But the line will only run when the position cannot be read from the peer configuration file.
 https://github.com/rustdesk/rustdesk/blob/master/flutter/lib/common.dart#L1662

So it should only run the first time when you connect to a new remote.
After closing the window, a new position should be written to the peer configuration file.
https://github.com/rustdesk/rustdesk/blob/master/flutter/lib/common.dart#L1541

What is the build date of your rustdesk?

![sss](https://github.com/rustdesk/rustdesk/assets/13586388/93e1f384-48bc-4be0-bc04-6e01d8702e19)


Please also find if `wm_RemoteDesktop` exists on your configuration file after closing the window.

![image](https://github.com/rustdesk/rustdesk/assets/13586388/8fe49c7f-7b04-4075-8b6e-0d6b4c21f145)

You can find the file here `%appdata%\rustdesk\config\peers\<peer id>.toml`

**Comment by Jemmota** [comment]
Thank you for your promptness in responding to my request. Thank you also to you and your team for the effort you have put into creating and maintaining such fantastic software as RustDesk.

Regarding the second problem, the screen displacement, strangely today I tried to reproduce it again and the problem did not occur anymore. The problem possibly arose because of the screen resolution reduction I made. My monitor is 4k and I use 150% image scaling. For some reason this fact may have contributed to this screen shift only occurring at that moment when I tested it.

However, if you can only solve the first problem, that would be great, since, in my opinion, I consider it to be the most important. My experience with the RustDesk code is very limited and I was not able to find the location where the window appears to try to solve the problem myself. Thank you one more time.

**Comment by Jemmota** [comment]
<img width="594" alt="rustdesk" src="https://github.com/rustdesk/rustdesk/assets/129885668/c0b20b1d-f850-432a-945a-76cd2446c3d9">

**Comment by Jemmota** [comment]
Dear @fufesou, today I noticed that there were updates to the RustDesk master repository. I did the merge and generated a new build of my fork. I carried out several tests and realized that the problem of white screens, which I had mentioned when opening the ticket, no longer occurred. Even using the executable generated by your Nightly version, I no longer noticed the problem. Apparently it has been resolved. If so, thank you very much.
